corehq/couchapps/last_modified/views/by_last_modified/map.js

```diff
@@ -0,0 +1,9 @@
+function(doc){
+    if (doc.doc_type === "Group" || doc.doc_type === "Group-Deleted") {
+        emit([doc.doc_type, doc.last_modified], null);
+    } else if (doc.doc_type === "Domain" || doc.doc_type === "Domain-Deleted") {
+        emit([doc.doc_type, doc.last_modified], null);
```

Comment:
something tells me i'm going to have to revisit this view... i think i'm going to opt to merge this now and when i touch this again, i will update to use a switch. there are a couple things i'd like to work on and PR independently that depend on this PR